07-contributor-guide.Rmd

```diff
@@ -38,515 +46,253 @@ Following good software development and coding practices simplifies collaboratio
+#### GitHub Projects
 
-## Issue Tracking
+[GitHub Projects](https://docs/github.com/en/issues/planning-and-tracking-with-projects) are used to keep track and prioritize issues. There are several [GitHub Projects for NOAA-FIMS](https://github.com/orgs/NOAA-FIMS/projects) such as the [Issue Triage Board](https://github.com/orgs/NOAA-FIMS/projects/21).
 
-Use of the GitHub issue tracker is key to keeping everyone informed and
-prioritizing key tasks. All future projects, ideas, concerns, development,
-etc. must be documented in an issue before the code is altered. Issues should
-be filed and tagged prior to any code changes whether the change pertains to a
-bug or the development of a feature. Issues are automatically tagged with the
-`status: triage_needed` tag and placed on the [Issue Triage
-Board](https://github.com/orgs/NOAA-FIMS/projects/21). Issues will subsequently
-be labeled and given an assignee and milestone by whoever is in charge of the
-Triage Board.
+### GitHub Cloning
 
-## Reporting Bugs
+Any repository within NOAA-FIMS can be [cloned](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository#cloning-a-repository) to a personal or virtual machine. But, contributors without write access will need to [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository) the repository and then clone their fork. 
 
-This section guides you through submitting a bug report for any toolbox tool. Following these guidelines helps maintainers and the community understand your report, reproduce the behavior, and find related reports.
+### GitHub Branching
 
-#### Before Submitting A Bug Report
+After cloning and before any work is done, ideas for potential changes to the code should be proposed in a [GitHub Issue](#github-issues). Next, after the issue is filed and before any work is done, a feature branch should be created where the work will be done.
 
-* **Check if it is related to version.** We recommend using `sessionInfo()` within your `R` console and submitting the results in your bug report. Also, please check your R version against the required R version in the DESCRIPTION file and update if needed to see if that fixes the issue.
-* **Perform a cursory search of issues** to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one. If it has **and the issue is closed**, open a new issue and include a link to the original issue in the body of your new one.
+#### Branch Naming Conventions
 
-#### How Do I Submit A (Good) Bug Report?
+1. Keep it brief
+1. Use a hyphen as separators
```

Comment:
Typo in the number? Should 1 be changed to 2, or are we relying on Markdown to automatically index the list?

Reply:
I am assuming that markdown will index the list.

Reply:
![image](https://github.com/NOAA-FIMS/collaborative_workflow/assets/4108564/33235142-193d-4e59-bf9d-2e04196ef171)
Confirmed that it does work.